docs/reference/release-notes/7.15.asciidoc

```diff
@@ -17,6 +17,9 @@ Authorization::
 Aggregations::
 * Possible source of leaked delayable writables {es-pull}80166[#80166]
 
+NOTE: The only arises when using <<search-search-api-query-params,allow_partial_search_results=false>> on a search with aggregations 
+AND if one of the underlying shard level search requests fail. Kibana uses `allow_partial_search_results=true` (Elasticsearch default).
```

Comment:
Transforms do use `allow_partial_search_results=true` and we know that it can provoke this issue. I think that if we clarify it does not happen in kibana, we also need to add that it is known that it can happen when using transforms.

Reply:
Sounds good, added :)